holoviews/core/spaces.py

```diff
@@ -817,6 +817,22 @@ def _cache(self, key, val):
         self.data[key] = val
 
 
+    def map(self, map_fn, specs=None, clone=True):
+        """
+        Recursively replaces elements using a map function when the
+        specification applies. Extends regular map with functionality
+        to dynamically apply functions.
+        """
```

Comment:
Ok. As long as you feel ``map`` is tested on sufficiently complex examples somewhere, I don't mind.